mantidproject/mantid
# Remove references to MantidPlot from Workbench

**Description of work.**
Some windows in Workbench had "MantidPlot" in their title, this has been changed to "Mantid".

**To test:**
One example of a window which has been changed:
1. Open the script editor and press ctrl+f.
2. In the "Find" input box, enter a string which is not present in the script.
3. Click the "Replace All" button.
4. The window which appears should have "Mantid" in its title instead of MantidPlot.

Fixes #25523 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
